guides/micronaut-cloud-database-base/end.adoc

```diff
@@ -0,0 +1 @@
+Read more about https://micronaut-projects.github.io/micronaut-data/latest/guide/[Micronaut Data] and https://micronaut-projects.github.io/micronaut-oracle-cloud/latest/guide/[Micronaut Oracle Cloud support].
```

Comment:
Could not we just have this in a common snippet name `micronaut-cloud-database-end`?